File: scratch/check_logs.js
#!/usr/bin/env node
require('dotenv').config();
const { Client } = require('ssh2');

const conn = new Client();
conn.on('ready', () => {
    conn.exec('docker logs skufia-api-green --tail 200 2>&1 | tail -60', (err, stream) => {
        if (err) { console.error(err); conn.end(); return; }
        let output = '';
        stream.on('data', d => { output += d.toString(); });
        stream.stderr.on('data', d => { output += d.toString(); });
        stream.on('close', () => {
            console.log(output);
            conn.end();
        });
    });
}).connect({
    host: '147.45.245.133',
    port: 22,
    username: 'root',
    password: process.env.SERVER_PASSWORD,
    readyTimeout: 10000
});

File: scratch/check_logs.py
import paramiko, sys
client = paramiko.SSHClient()
client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
client.connect('147.45.245.133', username='root', password='y38N*dQM.X33k?')

stdin, stdout, stderr = client.exec_command('''
cd /opt/skufia
docker compose -f docker-compose.production.yml logs --tail=50 backend-green
''')

out = stdout.read().decode()
err = stderr.read().decode()
print("=== BACKEND LOGS ===")
print(out)
if err:
    print("=== STDERR ===")
    print(err)
client.close()
